### ISSUE TITLE
<colgroup> doesn't detect closed tag when it doesn't see <col/>

### ISSUE THREAD

**Issue by pmcelhaney** [runem/lit-analyzer#161]
![image](https://user-images.githubusercontent.com/51504/108521198-7ac18e00-7299-11eb-92e2-4be9d8a5023f.png)

It looks like `<colgroup>` doesn't recognize the closing tag if it doesn't find a `<col/>` tag.  
```js
const works = html`<colgroup><col/></colgroup>`
const broken = html`<colgroup></colgroup>`
const also_broken = html`<colgroup>${myColTags}</colgroup>`
```
